src/Services/LoggingService.php

```diff
@@ -116,17 +125,29 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
-   * Maps data from the he associative array that Wayfair provides to the PlentyMarkets logging API's inputs
+   * Maps data from the associative array that Wayfair provides to the PlentyMarkets logging API's inputs
    *
    * @param $loggingInfo
    *
    * @return array
    */
   public function extractVars($loggingInfo): array {
+    $externalLogs = pluginApp(ExternalLogs::class);
+    $clientID = $this->configHelper->getClientId();
+    $shortMessage = [];
+
     $additionalInfo = $loggingInfo['additionalInfo'] ?? [];
     $method = $loggingInfo['method'] ?? null;
     $referenceType = $loggingInfo['referenceType'] ?? null;
     $referenceValue = (int) $loggingInfo['referenceValue'] ?? null;
+
+    if (strlen(json_encode($loggingInfo)) > self::STRING_LIMIT) {
+      $logForKibana = [];
+      $additionalInfo = $shortMessage['message'] = 'Message was too long to log in PlentyMarkets, ' . $clientID . '-' . date('M d Y H:i:s');
```

Comment:
ShortMessage is being used to set `additionalInfo` as an array, since an associative array is needed so the version and other necessary info can be added, and just setting `additionalInfo` to a string would give an conversion error

Reply:
order of operations is right to left, so `additonalInfo` would be set to a string here.